beacon_chain/validator_induction/src/inductor.rs

```diff
@@ -13,7 +13,7 @@ pub fn process_deposit(
     state: &mut BeaconState,
     deposit: &Deposit,
     spec: &ChainSpec,
-) -> Result<usize, ValidatorInductionError> {
+) -> Result<(), ValidatorInductionError> {
```

Comment:
How come you decided not to return the validator index anymore?

Reply:
They changed the specs so the return type is None.

My guess is that since they no longer remove validators that have been inactive for a certain period of time, therefore the deposit will always be input in the last index (rather than before it could take the index of the first validator to be removed for inactivity)

Reply:
Cool, makes sense. Thanks :)